src/Bridge/Symfony/Bundle/Resources/config/api.xml

```diff
@@ -154,6 +154,7 @@
             <argument type="service" id="api_platform.subresource_data_provider" />
             <argument type="service" id="api_platform.serializer.context_builder" />
             <argument type="service" id="api_platform.identifier.converter" />
+            <argument type="service" id="api_platform.metadata.resource.metadata_factory" on-invalid="null" />
```

Comment:
`on-invalid="null"` is useless, isn't it?

Reply:
Hmm... Indeed, `api_platform.metadata.resource.metadata_factory` must exist.